CHANGELOG.md

```diff
@@ -5,6 +5,26 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [5.1.0] - 2019-05-23
+
+### Added
+- UI: Accessibility - Make H1 readable by screen readers
+- UI: Accessibility - Make buttons/links readable by screen readers, allow tabbing to them
+- UI: Accessibility - Sort out order of items when tabbing through the content of each step
+- UI: Accessibility - Announce page transition when screen changes
+- UI: Accessibility - Make capture previews readable by screen readers
+- UI: Accessibility - Announce enlargement of captured image in preview
+- UI: Accessibility - Announce camera alerts
+- UI: Accessibility - Announce validation errors and warnings on confirm screen
+
+### Changed
+- Internal: Make Permission screen and Recovery screen buttons visible on small devices
```

Comment:
Forgot to include, please add:
`- Internal: The third party analytics (Woopra) is now imported via a dummy window in order not to pollute the shared global window`